airflow/www/static/js/connection_form.js

```diff
@@ -129,8 +154,9 @@ $(document).ready(() => {
   const testConnBtn = $('<button id="test-connection" type="button" class="btn btn-sm btn-primary" '
     + 'style="margin-left: 3px; pointer-events: all">Test\n <i class="fa fa-rocket"></i></button>');
 
+  // Disable the Test Connection button if Airflow REST APIs are not enabled.
   if (!restApiEnabled) {
-    $(testConnBtn).addClass('disabled')
+    $(testConnBtn).prop('disabled', true)
```

Comment:
We can delete line 142 and just do this:

```suggestion
    $("#testable_connection_types").prop('disabled', true)
```